roscala/src/test/scala/coop/rchain/rosette/TransitionSpec.scala

```diff
@@ -496,4 +496,27 @@ class TransitionSpec extends FlatSpec with Matchers {
     val end = VirtualMachine.executeSeq(codevec, start)
     end.ctxt.ctxt.rslt shouldBe Ob.RBLFALSE
   }
+
+  "OpRtnReg" should "copy ctxt.rslt to given register" in {
+
+    /**
+      * litvec:
+      *   0:   100
+      * codevec:
+      *   0:   liti 0,rslt
+      *   1:   rtn/reg/nxt
```

Comment:
Should be `* 1: rtn/reg/nxt 1`

Reply:
Or I guess `* 1: rtn/reg/nxt trgt`

Reply:
On further thought, it doesn't really make sense to specify the returning register of a codevec as that is the responsibility of the caller. Maybe

```
push/alloc 1
liti 0, rslt
rtn/reg/nxt, rslt
rtn/nxt
```
In other words the `rtn/reg/nxt, rslt` is effectively a no-op.

Reply:
As in the above is equivalent to if you did

```
liti 0, rslt
rtn/nxt
```